modules/oadp-checking-api-group-versions.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: PROCEDURE
+[id="oadp-checking-api-group-versions_{context}"]
+= Checking Kubernetes API group versions
+
+A source cluster might have a resource with multiple Kubernetes API versions registered to it. One of those versions is the preferred version. If you use Velero to back up and restore such a source cluster, Velero backs up only the version of that resource that uses the preferred version of its Kubernetes API.
+
+For example, if you have a resource that has 3 API versions - `v1alpha1`, `v1beta1`, and `v1` - registered on the cluster, Velero only backs up the version of that resource that uses the preferred API version of that CRD, v1. So, when Velero attempts to restore this resource, the target cluster must have the `v1` version registered in its set of available API resources.
```

Comment:
It's not really a resource that _has_ an API, exactly. Rather, the cluster offers different versions of an API, and that API lets you create some kind of resource.